mongo/document/document.go

```diff
@@ -22,14 +24,23 @@ type document interface {
 	Validable
 }
 
+var _ document = &Base{}
```

Comment:
No pointer

Reply:
The `Base` implementation of the `document` interface is on pointer. Cf. https://github.com/Scalingo/go-utils/pull/49/files#diff-9732770faadda6716a617078c80ccbe9L24